src/Nox.Docs/Templates/EntityEndpoints.template.md

```diff
@@ -11,6 +11,9 @@ This document provides information about the various endpoints available in our
 ### Get {{entity.PluralName}}
 - **GET** `/api/{{entity.PluralName}}`
   - Description: Retrieve information about {{entity.PluralName}}.
+{{~ if entity.IsLocalized ~}}
```

Comment:
Can you add those endpoints at the end within a section "Localized Endpoints" and with only one If statement which would wrap all endpoints?

Reply:
Desired changes have been applied